com.ca.lsp.cobol/lsp-service-cobol/src/test/java/com/ca/lsp/cobol/service/CopybookServiceE2ETest.java

```diff
@@ -19,55 +19,54 @@
 import com.broadcom.lsp.domain.cobol.event.model.RequiredCopybookEvent;
 import com.broadcom.lsp.domain.cobol.event.model.UnknownEvent;
 import com.ca.lsp.cobol.FileSystemConfiguration;
+import com.ca.lsp.cobol.model.ConfigurationSettingsStorable;
+import com.ca.lsp.cobol.service.delegates.communications.Communications;
+import com.ca.lsp.cobol.service.delegates.dependency.CopybookDependencyService;
+import com.ca.lsp.cobol.service.delegates.dependency.CopybookDependencyServiceImpl;
+import com.google.inject.Provider;
 import lombok.extern.slf4j.Slf4j;
 import org.awaitility.Duration;
 import org.awaitility.core.ConditionTimeoutException;
-import org.junit.After;
 import org.junit.Before;
 import org.junit.Test;
 
-import java.io.File;
-import java.io.IOException;
 import java.nio.file.Files;
-import java.nio.file.Path;
 import java.nio.file.Paths;
-import java.util.Comparator;
+import java.util.Arrays;
 
 import static org.awaitility.Awaitility.await;
 import static org.junit.Assert.assertEquals;
 import static org.junit.Assert.fail;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
 
 /**
  * This class contains all the unit test that perform the publish/subscribe acrivities for generate
  * the dependency file.
  */
 @Slf4j
-public class FileSystemE2ETest extends FileSystemConfiguration {
+public class CopybookServiceE2ETest extends FileSystemConfiguration {
   public static final String CPY_NAME_WITHOUT_EXT = "copy2";
-  DataBusBroker broker =
+  private DataBusBroker broker =
       (DefaultDataBusBroker) LangServerCtx.getInjector().getInstance(DataBusBroker.class);
```

Comment:
I cannot mock it, I need a real version because I need it for a unit test to create the dependency file

Reply:
I will check it - not sure at 100% I can mock it

Reply:
Right now you start up the context, but you don't tear it down. It would be better to call Guice directly to avoid dirty state here, e.g. 
```
  private final DataBusBroker databus =
      Guice.createInjector(new DatabusModule()).getInstance(DataBusBroker .class);
```

Reply:
👍